dotnet/samples/GettingStartedWithAgents/GettingStartedWithAgents.csproj

```diff
@@ -15,6 +15,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.AI.OpenAI" VersionOverride="2.0-beta.2" />
```

Comment:
Need to resolve version in one direction or the other.  Since Assistant functionality lives here, pinning to old version is a non-starter.  This also references the OpenAI memory connector.